exasol_advanced_analytics_framework/resources/outputs/create_query_loop.sql

```diff
@@ -12,380 +12,224 @@ CREATE OR REPLACE LUA SCRIPT "AAF_RUN_QUERY_HANDLER"(json_str) RETURNS TABLE AS
+--- Add issue ticket mitigation
 -- This is a special kind of mitigation which you should use in case of internal software errors that should not happen.
 -- For example when a path in the code is reached that should be unreachable if the code is correct.
--- </p>
---
 -- @return error message object
---
-function M:add_ticket_mitigation()
-    table.insert(self.mitigations,
+function ExaError:add_ticket_mitigation()
+    table.insert(self._mitigations,
         "This is an internal software error. Please report it via the project's ticket tracker.")
     return self
 end
 
-
----
--- Get the error code.
---
+--- Get the error code.
 -- @return error code
---
-function M:get_code()
-    return self.code
+function ExaError:get_code()
+    return self._code
 end
 
----
--- Get the error message.
--- <p>
--- This method supports Lua's standard string interpolation used in <code>string.format</code>.
+--- Get the error message.
 -- Placeholders in the raw message are replaced by the parameters given when building the error message object.
--- </p>
--- <p>
 -- For fault tolerance, this method returns the raw message in case the parameters are missing.
--- </p>
---
 -- @return error message
---
-function M:get_message()
-    return expand(self.message, self.parameters)
+function ExaError:get_message()
+    return expand(self._message, self._parameters)
 end
 
-function M:get_raw_message()
-    return self.message or ""
+function ExaError:get_raw_message()
+    return self._message or ""
 end
 
----
--- Get parameter definitions.
---
-function M:get_parameters()
-    return self.parameters
+--- Get parameter definitions.
+-- @return parameter defintions
+function ExaError:get_parameters()
+    return self._parameters
 end
 
----
--- Get the description of a parameter.
---
--- @parameter parameter_name name of the parameter
---
--- @return parameter description or the string "<code><missing parameter description></code>"
---
-function M:get_parameter_description(parameter_name)
-    return self.parameters[parameter_name].description or "<missing parameter description>"
+--- Get the description of a parameter.
+-- @param parameter_name name of the parameter
+-- @return parameter description or the string "`<missing parameter description>`"
+function ExaError:get_parameter_description(parameter_name)
+    return self._parameters[parameter_name].description or "<missing parameter description>"
 end
 
----
--- Get the mitigations for the error.
---
+--- Get the mitigations for the error.
 -- @return list of mitigations
---
-function M:get_mitigations()
-    return unpack(self.mitigations)
+function ExaError:get_mitigations()
+    return table.unpack(self._mitigations)
 end
 
----
--- Raise the error.
--- <p>
--- Like in Lua's <code>error</code> function, you can optionally specify if and from which level down the stack trace
+--- Raise the error.
+-- Like in Lua's `error` function, you can optionally specify if and from which level down the stack trace
```

Comment:
reformulate the sentence.

Reply:
Generated file which vendorizes all dependencies. This code is not part of our own codebase.